packages/rulesets/src/spectral/test/tracked-extension-resources-are-not-allowed.test.ts

```diff
@@ -0,0 +1,550 @@
+import { Spectral } from "@stoplight/spectral-core"
+import linterForRule from "./utils"
+
+let linter: Spectral
+
+beforeAll(async () => {
+  linter = await linterForRule("trackedExtensionResourcesAreNotAllowed")
+  return linter
+})
+
+test("trackedExtensionResourcesAreNotAllowed should find errors for tracked extension resources", () => {
+  const myOpenApiDocument = {
+    swagger: "2.0",
+    paths: {
+      "/{resourceUri}/providers/Microsoft.EdgeProvisioning/deviceProvisioningStates": {
```

Comment:
>/{resourceUri}/providers/Microsoft.EdgeProvisioning/deviceProvisioningStates [](http://example.com/codeflow?start=7&length=76)

Add tests for the following variants of extension resource api paths


Without the leading slash
{resourceUri}/providers/Microsoft.EdgeProvisioning/deviceProvisioningStates

Explicit extension 1
/subscriptions/{subid}/resourceGroups/{rgName}/providers/Microsoft.Foo/resourceType/{resourceInstance}/providers/Microsoft.EdgeProvisioning/deviceProvisioningStates


Explicit extension 2
/subscriptions/{subid}/providers/Microsoft.Foo/resourceType/{resourceInstance}/providers/Microsoft.EdgeProvisioning/deviceProvisioningStates


Explicit extension 3
/subscriptions/{subid}/providers/Microsoft.Foo/resourceType/{resourceInstance}/providers/Microsoft.Bar/resourceType/{resourceInstance}/providers/Microsoft.EdgeProvisioning/deviceProvisioningStates